### ISSUE TITLE
conda installable gin?

### ISSUE THREAD

**Issue by JuliaSprenger** [G-Node/gin-cli#208]
Since [git](https://anaconda.org/anaconda/git) and [git-annex](https://anaconda.org/conda-forge/git-annex) are both available as conda packages, is it possible to set up also a conda version of the gin client? This would simplify the integration of gin into workflows which are based on conda environments eg. due to restricted rights.

**Comment by cgars** [comment]
Hey all. 
I kind of believe we discussed this issue, however, I am unsure what the conclusion was? @JuliaSprenger  just reminded me of it and I thought its maybe useful to push this again. 

I would be of the opinion that Conda is kind of a strange way to ship go binaries (i know Conda says it's for any language... ) however I can kind of see (with quite some pain) that people tend to use it that way?

thoughts? or again did we agree on something?

**Comment by achilleas-k** [comment]
Since people use it for anything, I suppose it would be nice to make a G-Node user and upload gin-cli as well as other projects. As @JuliaSprenger mentioned, it will probably make it easier for anaconda users to get the gin client along with git-annex.

I'll update this issue with more info when I read up on the packaging guidelines for conda.

**Comment by morales-gregorio** [comment]
Hey all,
I would be very interested in having a conda installable gin-client, so I can use my environment in the cluster without having to install everything individually. Has there been progress on this front?

**Comment by achilleas-k** [comment]
I've been meaning to look into this for a while (over a year, by the looks it).  One issue is that I'm not sure how much work is involved so other things tend to take priority.  Does anyone know if this is as simple as creating a user on the Anaconda Cloud and uploading some packages?  Is the packaging and install script writing simple enough?

**Comment by JuliaSprenger** [comment]
I have no intuition for that. I think @alperyeg has more experience with that. Maybe he can tell you.

**Comment by alperyeg** [comment]
> Is the packaging and install script writing simple enough? 
Yes it is quite simple. Even simpler if you have a pip version. There are some basic steps you have to follow. Here is the landing page https://docs.conda.io/projects/conda-build/en/latest/ 
[git-annex](https://anaconda.org/conda-forge/git-annex) is on the anaconda cloud, probably it is easier to use that build as well.

**Comment by mpsonntag** [comment]
Comment by JuliaSprenger posted on gin: https://gin.g-node.org/G-Node/Info/issues/46

"As a temporary workaround I am currently manually adding the gin-cli to conda environments. A short summary of this procedure can be found here: https://gist.github.com/JuliaSprenger/ef009154d61716d85710772672749fbf"
